airflow/www/views.py

```diff
@@ -1084,6 +1084,28 @@ def blocked(self):
             })
         return wwwutils.json_response(payload)
 
+    @expose('/latest_dag_runs')
+    @login_required
+    def latest_dag_runs(self):
```

Comment:
I have no particular preference tbh. API feels a bit more natural as it is not just Data Access, but with some extra logic.